packages/x-date-pickers/src/DatePicker/DatePicker.tsx

```diff
@@ -12,6 +12,16 @@ type DatePickerComponent = (<TDate>(
   props: DatePickerProps<TDate> & React.RefAttributes<HTMLDivElement>,
 ) => React.JSX.Element) & { propTypes?: any };
 
+/**
+ * Demos:
+ *
+ * - [DatePicker](https://mui.com/x/react-date-pickers/date-picker/)
+ * - [Validation](https://mui.com/x/react-date-pickers/validation/)
+ *
+ * API:
+ *
```

Comment:
As far as I've tested, this new line is optional. Adding only the new line after `Demos` seems to solve the problem, but it won't hurt having it here as well. 👌